### DISCUSSION TITLE
How to give custom name virtual environments?

### DISCUSSION THREAD

**Discussion by Bugrakca** [JanDeDobbeleer/oh-my-posh#1170] [Troubleshoot]
Hello, I have been looking to give a custom name to virtual environments. I'm using Pipenv (Python) and the name is too long for me. I looked on the internet, didn't find anything for this. If is there a way to do this, please inform me.

Thank you.

**Comment by JanDeDobbeleer** [comment]
That's not a oh-my-posh responsibility but rather a pipenv thing. Rename the environment correctly, [this](https://stackoverflow.com/a/43396657) guide has some information.

**Reply by Bugrakca** [reply]
I thought maybe there is a way not to show the full name. But thank you so much for the clarification.

**Reply by JanDeDobbeleer** [reply]
There will be when I add template support! Stay tuned!

**Reply by Bugrakca** [reply]
When is it possible to update the docs for Template support?

**Reply by JanDeDobbeleer** [reply]
@Bugrakca this hasn't been done just yet ;-)

**Reply by JanDeDobbeleer** [reply]
@Bugrakca [Python](https://ohmyposh.dev/docs/python#template-properties) now has the `.Venv` template property, you can make use of sprig/template functions to play with it.

**Reply by Bugrakca** [reply]
Thank you 👍
